base/java-tools/src/com/netscape/cmstools/authority/AuthorityKeyExportCLI.java

```diff
@@ -45,10 +50,18 @@ public AuthorityKeyExportCLI(AuthorityCLI authorityCLI) {
         option = new Option(null, "target-nickname", true, "Nickname of target key");
         option.setArgName("nickname");
         options.addOption(option);
+
+        option = new Option(null, "algorithm", true, "Symmetric encryption algorithm");
+        option.setArgName("OID");
+        options.addOption(option);
     }
 
     public void printHelp() {
-        formatter.printHelp(getFullName() + "--wrap-nickname NICKNAME --target-nickname NICKNAME -o FILENAME", options);
+        formatter.printHelp(
+            getFullName()
+                + "--wrap-nickname NICKNAME --target-nickname NICKNAME -o FILENAME"
+                + "[--algorithm OID]",
```

Comment:
I think we need to add a space before `--wrap-nickname NICKNAME` and before `[--algorithm OID]`.

Reply:
Thanks, I'll fix this in the push.